pkg/kubelet/eviction/helpers.go

```diff
@@ -83,6 +86,7 @@ func init() {
 	signalToResource[evictionapi.SignalImageFsInodesFree] = resourceImageFsInodes
 	signalToResource[evictionapi.SignalNodeFsAvailable] = resourceNodeFs
 	signalToResource[evictionapi.SignalNodeFsInodesFree] = resourceNodeFsInodes
+	signalToResource[evictionapi.SignalAllocatableStorageScratchAvailable] = resourceNodeFs
 	resourceToSignal = map[v1.ResourceName]evictionapi.Signal{}
```

Comment:
Yes, I will do it in a separate PR, basically changing from 1-1 mapping from resource to signal, we use map 1-many with a list of signals